.github/workflows/deploy-k8s.yml

```
@@ -0,0 +1,78 @@
+env:
+  REGISTRY: crretoxmas2024.azurecr.io
+  IMAGE_NAME: ithaka-backend
+  NAMESPACE: ithaka-backend
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'  # Solo en main
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)